src/pages/workspaces/workspace/Data.js

```diff
@@ -688,13 +692,23 @@ const WorkspaceData = _.flow(
                       onToggleVersionHistory: withErrorReporting('Error loading version history', showVersionHistory => {
                         setShowDataTableVersionHistory(_.set(type, showVersionHistory))
                         if (showVersionHistory) {
-                          loadDataTableVersions(type)
+                          loadDataTableVersions(type.endsWith('_set') ? _.replace(/(_set)+$/, '', type) : type)
                         }
                       })
                     })
                   }),
                   isShowingVersionHistory && h(DataTableVersions, {
-                    ...dataTableVersions[type],
+                    ...Utils.cond(
+                      [type.endsWith('_set'), () => {
+                        const referencedType = _.replace(/(_set)+$/, '', type)
```

Comment:
`_.replace(/(_set)+$/, '', type)` seems to be in a few different places. Is it work making it a utility method?

Reply:
Done in [f2838c5](https://github.com/DataBiosphere/terra-ui/pull/3434/commits/f2838c53ada51d537fa28ec99aa5a5068d03535d).